src/openms/source/KERNEL/Feature.cpp

```diff
@@ -234,4 +234,13 @@ namespace OpenMS
     subordinates_ = rhs;
   }
 
+  void Feature::updateAllIDReferences(const IdentificationData::RefTranslator& trans)
+  {
+    updateIDReferences(trans);
```

Comment:
can you add some comment here so it is easier to understand why updateID and updateAllID are used...